apps/user-office-backend/src/resolvers/types/ProposalView.ts

```diff
@@ -87,3 +98,23 @@ export class ProposalView implements Partial<ProposalOrigin> {
   @Field(() => AllocationTimeUnits)
   public allocationTimeUnit: AllocationTimeUnits;
 }
+
+@Resolver((of) => ProposalView)
```

Comment:
@RasmiaKulan I have a small comment here. I think in the `ProposalView` we should avoid using the resolvers because the purpose of the view is to get combined tables data much quicker. Probably the right way to do it is to update the view containing the needed information and then just use that one directly. Here is an example: https://github.com/UserOfficeProject/user-office-core/blob/develop/apps/user-office-backend/db_patches/0097_AddAllocationTimeUnitToProposalViewTable.sql. You should delete the view and recreate it with the new columns needed.